airflow/gcp/hooks/bigquery.py

```diff
@@ -1430,6 +1430,39 @@ def cancel_query(self) -> None:
                               self.running_job_id)
                 time.sleep(5)
 
+    def get_dataset_tables(self, dataset_id: str, project_id: Optional[str] = None,
+                           max_results: Optional[int] = None,
+                           page_token: Optional[str] = None) -> Dict[str, Union[str, int, List]]:
+        """
+        Get the list of tables for a given dataset.
+        see https://cloud.google.com/bigquery/docs/reference/rest/v2/tables/list
```

Comment:
```suggestion

        .. seealso:: https://cloud.google.com/bigquery/docs/reference/rest/v2/tables/list
```
Looks better imo. Don't you think so? :)

https://www.sphinx-doc.org/en/1.5/markup/para.html#directive-seealso

Reply:
Totally agreed. 👍